src/configs/app.config.js

```diff
@@ -24,7 +24,8 @@ const languages = {
     'ta': 'தமிழ்',
     'th-TH': 'ไทย',
     'tr-TR':'Türkçe',
-    'zh-TW': '繁體中文'
+    'zh-TW': '繁體中文',
+    'uk-UA': 'Українська',
```

Comment:
```suggestion
    'uk-UA': 'Українська',
    'zh-TW': '繁體中文'
```
Keep the list sorted by language code.